app/Http/Controllers/ArticleController.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\ArticleRequest;
+use App\Models\Article;
+use App\Repositories\ArticleRepository;
+use App\Repositories\CategoryRepository;
+use App\Repositories\TagRepository;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Log;
+
+class ArticleController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index(ArticleRepository $articleRepository)
```

Comment:
@gemaantikahr it's a personal opinion, but in my experience, repositories are overcomplicating the projects if they are used just for this.

Watch my video on this: https://www.youtube.com/watch?v=giJcdfW2wC8